usaspending_api/api_docs/api_documentation/advanced_award_search/spending_by_transaction.md

```diff
@@ -0,0 +1,82 @@
+## [Spending by Transaction](#spending-by-transaction)
+**Route:** `/api/v2/search/spending_by_transaction/`
+
+**Method:** `POST`
+
+This route takes keyword search fields, and returns the fields of the searched term.
+
+### Request
+field: Defines what award variables are returned.
+
+filter: Defines how the transactions are filtered.  Search term and pannel-name are required. Transaction type is one of pane they are looking at. Can be one of the following- ["Contracts", "Grants", "Direct Payments", "Loans", "Other"]
+
+https://github.com/fedspendingtransparency/usaspending-api/wiki/Search-Filters-v2-Documentation
+
+limit(**OPTIONAL**): how many results are returned
+
+page (**OPTIONAL**):  what page of results are returned
+
+sort (**OPTIONAL**): Optional parameter indicating what value results should be sorted by. Valid options are any of the fields in the JSON objects in the response. Defaults to the first `field` provided.
+
+order (**OPTIONAL**): Optional parameter indicating what direction results should be sorted by. Valid options include `asc` for ascending order or `desc` for descending order. Defaults to `asc`.
+
+```
+{
+  "filters": {
+       "keyword": "Booz Allen",
+       "award_type_codes" : ["10"],
+       },
+  "fields": ["Award ID", "Recipient Name", "Action Date", "Transaction Amount", "Awarding Agency", "Awarding Sub Agency", "Award Type", "Funding Agency", "Funding Sub Agency", "Mod"],
+  "sort": "Recipient Name",
+  "order": "desc",
+  "page": 1,
+  "limit": 30,
+}
+```
+
+### Response (JSON)
+
+```
+{
+    "limit": 10,
+    "results": [
+        {
+            "internal_id": "9359566,
+            "Award ID": "9359566"",
```

Comment:
This suggests that `Award ID` is the same value as `internal_id`. Document that `internal_id` is the internal API identifier; `Award ID` is an actual data element.

Also, use a different value for Award ID, like `W15P7T06CN401`

Reply:
Done